rust_src/src/editfns.rs

```diff
@@ -936,4 +941,84 @@ pub fn buffer_string() -> LispObject {
     unsafe { make_buffer_string_both(begv, begv_byte, zv, zv_byte, true) }
 }
 
+// Save current buffer state for `save-excursion' special form.
+// We (ab)use Lisp_Misc_Save_Value to allow explicit free and so
+// offload some work from GC.
```

Comment:
Oh.

You need the `#[no_mangle]`. This tells Rust to export the name as is which is what C expects.

Reply:
Did my tool leave it off?

Reply:
Ah, that must be it. I didn't use the tool for this one, but I guess I should have.